### PR TITLE
[IMP] accounting: bank transactions

### PR DESCRIPTION
**Pull request by chiaraprattico**
taskid 3204835

### CODE REVIEW

File: content/applications/finance/accounting/bank/feeds/bank_statements.rst [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
 
-Importing your bank statements allows you to keep track of your bank account transactions and
-reconcile them with the ones recorded in Odoo.
+Importing bank statements allows keeping track of bank account transactions and reconcile them with
+the ones recorded in Odoo.
 
-We recommend using :doc:`the bank synchronization feature <bank_synchronization>` for more
-efficiency. However, if you do not want to use it or if your bank is not yet supported, other
+With :doc:`the bank synchronization feature <bank_synchronization>` the process is going to be
+automated. However, if you do not want to use it or if your bank is not yet supported, other
```
Comment (Donapi) [inline content/applications/finance/accounting/bank/feeds/bank_statements.rst:9]:
```suggestion
With :doc:`bank synchronization <bank_synchronization>`, the process is
automated. However, if you do not want to use it or if your bank is not yet supported, other
```

File: content/applications/finance/accounting/bank/feeds/bank_statements.rst [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
 options exist:
 
 - Import the bank statement files delivered by your bank;
```
Comment (Donapi) [inline content/applications/finance/accounting/bank/feeds/bank_statements.rst:12]: - Import bank statement files;
- Register bank transactions manually.
- --> I would take the title of the sections below and add custom anchors.

File: content/applications/finance/accounting/bank/feeds/bank_statements.rst [hunk 3]
```diff
@@ -23,8 +23,8 @@ Odoo supports multiple file formats to import bank statements:
 - Quicken Interchange Format (.QIF);
 - Belgium: Coded Statement of Account (.CODA).
 
-To import a file, go to the Accounting dashboard and click on :guilabel:`Import (Statements)` under
-the :guilabel:`Bank` window, or click on the drop-down menu (:guilabel:`⋮`) button and then on
+To import a file, go to the Accounting dashboard and click on :guilabel:`Import (Statements)` in the
+the :guilabel:`Bank` journal, or click on the drop-down menu (:guilabel:`⋮`) button and then on
```
Comment (Donapi) [inline content/applications/finance/accounting/bank/feeds/bank_statements.rst:27]:
```suggestion
To import a file, go to the :guilabel:`Accounting Dashboard` and in the
the :guilabel:`Bank` journal, click on :guilabel:`Import Statement`. 
```

File: content/applications/finance/accounting/bank/feeds/bank_statements.rst [hunk 4]
```diff
@@ -33,30 +33,19 @@ the :guilabel:`Bank` window, or click on the drop-down menu (:guilabel:`⋮`) bu
 
 Next, select the file you want to import and click on :guilabel:`Upload`.
```
Comment (Donapi) [inline content/applications/finance/accounting/bank/feeds/bank_statements.rst:34]: I don't see the "upload" option. I would just say "Next, select and upload the file you want".

File: content/applications/finance/accounting/bank/feeds/bank_statements.rst [hunk 5]
```diff
@@ -33,30 +33,19 @@ the :guilabel:`Bank` window, or click on the drop-down menu (:guilabel:`⋮`) bu
...
 
-Once the file uploaded, the **import widget** helps you set formatting options and **map** the
-different columns you want to import. You also can :guilabel:`test` the bank statement file before
-importing it to your database.
+Once the file is uploaded, after setting the necessary formatting options and mapping the needed
+columns, click on :guilabel:`import`. It is also possible to :guilabel:`test` the bank statement
+file before importing it.
```
Comment (Donapi) [inline content/applications/finance/accounting/bank/feeds/bank_statements.rst:38]:
```suggestion
After setting the necessary formatting options and mapping the needed
columns, you have the possibility to make a :guilabel:`Test` or you can directly :guilabel:`Import` your bank statement files.
```

**Review by Donapi — CHANGES_REQUESTED** [review]
Hi Chiara, please find some suggestions :) we can review them together if you want.

**Review by Donapi — APPROVED** [review]
Good work Chiara!

File: content/applications/finance/accounting/bank/feeds/bank_statements.rst [hunk 6]
```diff
@@ -23,40 +24,26 @@ Odoo supports multiple file formats to import bank statements:
...
-different columns you want to import. You also can :guilabel:`test` the bank statement file before
-importing it to your database.
+Next, select the file you want to import and upload it.
 
-.. image:: bank_statements/import-bank-statement.png
-   :align: center
-   :alt: Register bank statements manually in Odoo Accounting
+After setting the necessary formatting options and mapping the needed columns, you have the
+possibility to make a :guilabel:`Test` or you can directly :guilabel:`Import` your bank statement
+files.
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/feeds/bank_statements.rst:38]:
```suggestion
After setting the necessary formatting options and mapping the needed columns, you can
make a :guilabel:`Test` or directly :guilabel:`Import` your bank statement
files.
```
Suggestion to shorten/simplify

File: content/applications/finance/accounting/bank/feeds/bank_statements.rst [hunk 7]
```diff
@@ -23,40 +24,26 @@ Odoo supports multiple file formats to import bank statements:
...
    :align: center
-   :alt: Import a bank statement file in Odoo Accounting
-
-Next, select the file you want to import and click on :guilabel:`Upload`.
+   :alt: Import bank statements from bank journal
 
-Once the file uploaded, the **import widget** helps you set formatting options and **map** the
-different columns you want to import. You also can :guilabel:`test` the bank statement file before
-importing it to your database.
+Next, select the file you want to import and upload it.
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/feeds/bank_statements.rst:34]:
```suggestion
Next, select the file and upload it.
```
Suggestion to shorten/simplify

**Review by xpl-odoo — APPROVED** [review]
One spelling mistake and a couple of suggestions to shorten the text.
Otherwise, already approved ;)

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
@xpl-odoo @chiaraprattico 
I think we're missing Odoo 16's new paradigm:
Bank statements are not the focus anymore, while bank transactions are.
(It is even renamed "import transactions" from Odoo 16.2 :) )
image


It's true that we can still import bank statements manually, and all transactions would be part of a "statement" in Odoo too. Otherwise, I think the whole doc should be about "bank transactions" (also the title and filename).

**Review by jcs-odoo — APPROVED** [review]
@robodoo r+

**Comment by jcs-odoo** [comment]
@robodoo r-

**Comment by jcs-odoo** [comment]
@robodoo r+

**Pull request closed without merging**